.rubocop.yml

```diff
@@ -59,3 +59,8 @@ Metrics/LineLength:
 # It's not possible to set TargetRubyVersion to Ruby < v2.2
 Gemspec/RequiredRubyVersion:
   Enabled: false
+
+Style/BracesAroundHashParameters:
+  Exclude:
+    # we specifically want to test hash parameters
+    - 'test/acceptance/keyword_arguments_test.rb'
```

Comment:
Minor: I think I would prefer this to be disabled in the actual file i.e. around the whole test case if necessary. In my experience, these file-level exclusions tend to get missed/forgotten about over time.